scanners/googleanalytics.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import os
+import json
+import time
+
+from scanners import utils
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+command = None
+init = None
+
+
+class GAChecker:
+
+    def __init__(self):
+        d = DesiredCapabilities.PHANTOMJS
+        d['loggingPrefs'] = {'browser': 'ALL'}
+        self.browser = webdriver.PhantomJS(desired_capabilities=d)
+
+    def get_url(self, url):
+        """ Get URL fron browser """
+        self.browser.get(url)
+        time.sleep(3)
+
+    def run_checker_script(self):
+        """ Runs the GA checker script """
+        self.browser.execute_script("""
+if (typeof ga === "function") {
+    console.log('ga_version: Google Analytics Universal');
+    console.log('ga_ua_code: ' + ga.getAll()[0].get('trackingId'));
+    console.log('ga_anon_ip: ' + ga.getAll()[0].get('anonymizeIp'));
+    console.log('ga_force_ssl: ' + ga.getAll()[0].get('forceSSL'));
+}
+else if (typeof _sendPageview === "function") {
+    console.log('ga_version: Google Analytics Legacy');
+    console.log('ga_ua_code: ' + _gat._getTrackerByName()._getAccount());
+    console.log('ga_anon_ip: Google Analytics Legacy');
+    console.log('ga_force_ssl: Google Analytics Legacy');
+}
+else {
+    console.log('ga_version: No Google Analytics');
+    console.log('ga_ua_code: No Google Analytics');
+    console.log('ga_anon_ip: No Google Analytics');
+    console.log('ga_force_ssl: No Google Analytics');
+}
+        """)
+
+    def clean_message(self, element, message):
+        """ Cleans the message log """
+        return message.split(element)[-1].replace(' (:)', '').strip(': ')
```

Comment:
Instead of parsing the message using colons, the JavaScript should make a JSON object and use a single `console.log` command to spit it out. This will make parsing safer, more reliable, and will map directly to a Python dict.